README.md

```diff
@@ -49,10 +57,11 @@ Using Ruby, remove Belle from the town residents, and
 add her to the list of guests in the castle.
 
 Write your code here:
-```ruby
-# code here
-```
-
+residents.remove ("Belle")
```

Comment:
`.remove` is not a method you can perform on a Ruby array. You should use `delete`.

Also, `residents` belongs to the `town` hash, meaning you need to drill into the hash in order to access `residents`, like so...

```rb
town[:residents]
```